doc/source/dev/roadmap-detailed.rst

```diff
@@ -413,12 +413,12 @@ may also be included in SciPy, especially if no other widely used and
 well-supported package covers the topic.  Also note that *some* duplication
 with downstream projects is inevitable and not necessarily a bad thing.)
 
-The following improvements will help SciPy better serve this role.
+In addition to the items described in the :ref:`scipy-roadmap`, the following
+improvements will help SciPy better serve this role.
 
 - Add fundamental and widely used hypothesis tests:
 
-  - Tukey-Kramer test
-  - Dunnett's test
+  - Dunnett's test and the Student–Newman–Keuls test
```

Comment:
Tukey-Kramer was added. Student–Newman–Keuls is another multiple-comparison test that makes use of the studentized range distribution.